sdk/cognitivelanguage/Azure.AI.Language.Conversations/samples/Sample5_AnalyzeConversation_ConversationPii.md

```diff
@@ -0,0 +1,100 @@
+# Analyze a conversation with Conversation Summarization
+
+This sample demonstrates how to detect and redact personally identifiable information from a conversation with Conversation Pii. To get started, you'll need to create a Cognitive Language service endpoint and an API key. See the [README](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cognitivelanguage/Azure.AI.Language.Conversations/README.md) for links and instructions.
+
+Start by importing the namespace for the `ConversationAnalysisClient` and related classes:
+
+```C# Snippet:ConversationAnalysisClient_Namespaces
+using Azure.Core;
+using Azure.Core.Serialization;
+using Azure.AI.Language.Conversations;
+```
+
+To analyze an utterance, you need to first create a `ConversationAnalysisClient` using an endpoint and API key. These can be stored in an environment variable, configuration setting, or any way that works for your application.
+
+```C# Snippet:ConversationAnalysisClient_Create
+Uri endpoint = new Uri("https://myaccount.cognitiveservices.azure.com");
+AzureKeyCredential credential = new AzureKeyCredential("{api-key}");
```

Comment:
Have we considered a doc showing how to do this without keys (e.g. Managed Identity)/with 1keyv2, etc

Reply:
We definitely have one with AAD. It might be at the README. let me check.

Reply:
https://github.com/Azure/azure-sdk-for-net/tree/07243563fcdf1aa9b19f88e6b2952fb111d232ba/sdk/cognitivelanguage/Azure.AI.Language.Conversations#create-a-client-using-azure-active-directory-authentication

Reply:
discussed already. Its in the main readme. Not in all samples.